playbooks/provisioning/openstack/post-provision-openstack.yml

```diff
@@ -1,70 +1,93 @@
 ---
+- hosts: cluster_hosts
+  name: Wait for the the nodes to come up
+  become: False
+  gather_facts: False
+  tasks:
+    - ping:
```

Comment:
Ping is going to fail if the hosts are not up, isn't it? This should be moved to later (but why is it necessary at all?).